scst-store/failover.hbs.md

```diff
@@ -10,10 +10,12 @@ Users have the option to configure the number of replicas using the `app_replica
 
 ###  <a id="database"></a>Database
 
-By default, the database  has 1 replica, and  restarts with some downtime if it were to fail.
+By default, the database  has 1 replica, and  restarts with some downtime if it were to fail. Although the field `db_replicas` exists and is configurable by the user in the `scst-store-values.yaml` file, VMware discourages using it.
```

Comment:
To me it feels strange that it reads 'VMWare discourages using it'. Is it VMware or us?  I feel it should be replaced with 'It is discouraged from using it."

Also, should we explain why?  Maybe this phrase should be worded more like "...it is strongly discouraged to have more than one replica for the following reasons..."